src/porepy/models/solution_strategy.py

```diff
@@ -138,6 +138,9 @@ def __init__(self, params: Optional[dict] = None):
         self.solid: pp.SolidConstants
         """Solid constants. See also :meth:`set_materials`."""
 
+        self.num: pp.NumericalConstants
+        """Numerical constants. See also :meth:`set_materials`."""
+
```

Comment:
I would opt for `numerical` instead of `num` matching the style of `solid` and `SolidConstants` (similar for `fluid`).

Reply:
Definitely. "num" refers to "number" in PorePy parlance.